mlreco/models/layers/gnn/factories.py

```diff
@@ -142,8 +142,6 @@ def node_encoder_dict():
 
     encoders = {
         "geo"       : geometric.ClustGeoNodeEncoder,
-        "geo_norm"  : geometric.NormedClustGeoNodeEncoder,
```

Comment:
Is this just completely obsolete by now? Can you clarify what it was/who was using it/why we remove it?

Reply:
I think we agreed to just retrain the network on a larger ~6k^3 volume to resolve the spatial size dependency issue. Since I was the only one playing with it I just removed it

Reply:
hmm, ok, I will clarify this with @francois-drielsma. My understanding was that we would do both, but I may very well misremember.

Reply:
Francois confirmed your memory that we are doing the retraining only for now, but he added that we might look more into spatial-size-agnostic features in the future? I am fine with removing this if you have no plans to extend it, but it's also fine to leave it for future reference.